src/Components/UI/Navigation/DesktopNavigationBar.jsx

```diff
@@ -22,13 +29,29 @@ const DesktopNavigationBar = () => {
     setSearchInput(e.target.value);
   };
 
+  const onWishlistClickHandler = () => {
+    if (!auth.login) {
+      setShowLoginModal(true);
+    }
+  };
+
+  const onCartClickHandler = () => {
+    if (!auth.login) {
+      setShowLoginModal(true);
+    }
+  };
+
+  const dp = auth.user.dp !== "" ? auth.user.dp.toUpperCase() : "";
+  const cartBadgeValue = auth.login ? cartState.cart.length : 0;
+  const wishlistBadgeValue = auth.login ? wishlist.length : 0;
```

Comment:
Suggestion: Instead of showing 0 you can hide it

Reply:
Fixed